packages/bsky/src/data-plane/server/indexing/plugins/post.ts

```diff
@@ -175,6 +175,37 @@ const insertFn = async (
       }
       embeds.push(recordEmbed)
       await db.insertInto('post_embed_record').values(recordEmbed).execute()
+
+      const quote = {
+        uri: uri.toString(),
+        cid: cid.toString(),
+        subject: record.uri,
+        subjectCid: record.cid,
+        createdAt: normalizeDatetimeAlways(obj.createdAt),
+        indexedAt: timestamp,
+      }
+      await db
+        .insertInto('quote')
+        .values(quote)
+        .onConflict((oc) => oc.doNothing())
+        .returningAll()
+        .executeTakeFirst()
```

Comment:
Perhaps we only want to index these quotes when `record` is a post.

Reply:
Might need a refresher haha, but isn't that what `isEmbedRecord()` will ensure?

Reply:
I believe `isEmbedRecord()` ensures that a record is being embedded, but that record isn't necessarily a post— could also be e.g. a list, feedgen, or starter pack.  And I assume we only want embedded posts to be considered QPs.